debian/rules

```diff
@@ -8,6 +8,10 @@ export PYBUILD_NAME=eos-data-distribution
 
 override_dh_install:
 	dh_install --fail-missing
+	mkdir -p debian/edd-avahi-monitor/usr/bin
```

Comment:
@dbnicholson Is it typical to have to add this customization to debian/rules when splitting into multiple packages, or is there a better way?